docs/README.md

```diff
@@ -0,0 +1,40 @@
+# COMPOSITE BUILDS
```

Comment:
What is the title for the page, "Developer Documentation"? If so, I would suggest doing it explicitly

Reply:
Maybe it also makes sense to use `docs/developer.md` so that you reserve `docs/README.md` for the documentation root